File: 08_Inverted full pyramid/inverted_full_pyramid.py
rows = int(input("Enter Rows: "))
itr = rows

for i in range(0, rows):
    for j in range(0, i):
        print(" ",end=" ")
    
    for k in range(0, itr):
        print(" i ", end=" ")
    
    itr = itr-1
    print("")

'''
OUTPUT:

 *   *   *   *   *   *   *   *
   *   *   *   *   *   *   *
     *   *   *   *   *   *
       *   *   *   *   *
         *   *   *   *
           *   *   *
             *   *
               *
'''

File: 08_Inverted full pyramid/inverted_full_pyramid.c
#include<stdio.h>

int main(int argc, char const *argv[])
{
    int i,j,k,itr,rows;

    printf("Enter Rows: ");
    scanf("%d", &rows);

    itr = rows;

    for(i=0; i<rows; i++){
        for(j=0; j<i; j++){
            printf("  ");
        }
        for(k=0; k<itr; k++){
            printf(" *  ");
        }
        --itr;
        printf("\n");
    }
    return 0;
}

/*
OUTPUT:

 *   *   *   *   *   *   *   *
   *   *   *   *   *   *   *
     *   *   *   *   *   *
       *   *   *   *   *
         *   *   *   *
           *   *   *
             *   *
               *
*/